pkgs/development/python-modules/particle/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
+, setuptools-scm
+, attrs
+, deprecated
+, hepunits
+, pytestCheckHook
+, pytest-benchmark
+, pytest-cov
+, tabulate
+, pandas
+}:
+
+buildPythonPackage rec {
+  pname = "particle";
+  version = "0.20.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-HoWWwoGMrkRqlYzrF2apGsxsZAHwHbHSO5TCSCelxUc=";
+  };
+  nativeBuildInputs = [
+    # Otherwise version 0.0.0 is written to .dist-info directory and no error
+    # is shown.
+    setuptools-scm
+  ];
+
+  propagatedBuildInputs = [
+    attrs
+    deprecated
+    hepunits
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+    pytest-benchmark
+    pytest-cov
```

Comment:
```suggestion
```
we don't want to use resources for benchmarks or generate cov reports.

Reply:
> we don't want to use resources for benchmarks or generate cov reports.

But otherwise the build fails:

```
         > ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
         > __main__.py: error: unrecognized arguments: --benchmark-disable
         >   inifile: /build/particle-0.20.1/pyproject.toml
         >   rootdir: /build/particle-0.20.1
```

Reply:
You want something like:

``` 
postPatch = ''
  substituteInPlace pyproject.toml \
    --replace '"--benchmark-disable", ' ""
'';
```